From bcb73015a1d01c39d8f4d2825ee7d6a5a16693c5 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Tue, 8 Jan 2013 15:56:03 +0800
Subject: [PATCH 04/34] ENGR00239187 build: fix error bring by last ubi
 support patch.

This patch is fixing the ext4 image generate to yaffs2 image issue.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 tools/releasetools/build_image.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/releasetools/build_image.py b/tools/releasetools/build_image.py
index af5020c..784ad4e 100755
--- a/tools/releasetools/build_image.py
+++ b/tools/releasetools/build_image.py
@@ -297,7 +297,7 @@ def BuildImage(in_dir, prop_dict, out_file, target_out=None):
   elif fs_type.startswith("f2fs"):
     build_command = ["mkf2fsuserimg.sh"]
     build_command.extend([out_file, prop_dict["partition_size"]])
-  if fs_type.startswith("ubifs"):
+  elif fs_type.startswith("ubifs"):
     # add ubifs image generate support.
     build_command = ["mkfs.ubifs", "-d"]
     build_command.append(in_dir)
-- 
1.8.0

